pulpcore/cli/container/context.py

```diff
@@ -32,13 +32,17 @@ class PulpContainerDistributionContext(PulpEntityContext):
     CREATE_ID = "distributions_container_container_create"
     UPDATE_ID = "distributions_container_container_partial_update"
     DELETE_ID = "distributions_container_container_delete"
+    NULLABLES = {"repository_version", "repository"}
 
     def preprocess_body(self, body: EntityDefinition) -> EntityDefinition:
         body = super().preprocess_body(body)
         version = body.pop("version", None)
         if version is not None:
             repository_href = body.pop("repository")
             body["repository_version"] = f"{repository_href}versions/{version}/"
+        if not self.pulp_ctx.has_plugin(PluginRequirement("container", min="2.10.0.dev")):
+            if body.get("repository_version") == "":
+                del body["repository_version"]
```

Comment:
Now that i think of this, at this point, line 38 would have transformed the `""` into `None` already, since you specified `"repository_version"` in `NULLABLES`.